src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write code here
+const ul = document.querySelector('ul');
+const lists = [...ul.children];
+
+function sortList(list) {
+  const sortedList = list.sort((a, b) =>
+    Number(b.dataset.salary.split('$').join('').split(',').join('')
+    - Number(a.dataset.salary.split('$').join('').split(',').join('')))
```

Comment:
Create a function for converting to a number and use it for `a` and `b`. Now you use the same logic twice